packages/windows/vcruntime/vs2017/redist.txt

```diff
@@ -0,0 +1,14 @@
+Subject to Microsoft Licensing Product Use Rights [1], This package
+contains Visual C++ Runtime files as redistributable code:
+
+[1] http://download.microsoft.com/download/7/8/6/786cbfca-6826-4c58-9f4c-ee2067e619f5%2FPUR0604.pdf

```

Comment:
For VC++ 2017 redistributables, we can use this documentation:
https://visualstudio.microsoft.com/license-terms/mlt552233/